openapi-specs/nomis-prisoner-api-docs.json

```diff
@@ -13177,7 +13177,7 @@
           "required" : false,
           "schema" : {
             "type" : "array",
-            "default" : [ "createDatetime,ASC" ],
+            "default" : [ "createDatetime,ASC", "agencyVisitTimesId.location,ASC", "agencyVisitTimesId.weekDay,ASC", "agencyVisitTimesId.timeSlotSequence,ASC" ],
             "items" : {
```

Comment:
interestingly this ends up as a default parameter on generated code - hence you need this change to get the new sort order